nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -51,6 +52,41 @@ collision_monitor:
       min_points: 6
       visualize: False
       enabled: True
+    VelocityPolygonApproach:
+      type: "velocity_polygon"
+      action_type: "approach"
+      time_before_collision: 2.0
+      simulation_time_step: 0.1
+      min_points: 6
+      visualize: True
+      enabled: True
+      polygon_pub_topic: "polygon_approach"
+      velocity_polygons: ["rotation", "translation_forward", "translation_backward", "stop"]
+      holonomic: false
+      rotation:
+        points: "[[0.3, 0.3], [0.3, -0.3], [-0.3, -0.3], [-0.3, 0.3]]"
+        linear_min: 0.0
+        linear_max: 0.05
+        theta_min: -1.0
+        theta_max: 1.0
+      translation_forward:
+        points: "[[0.35, 0.3], [0.35, -0.3], [-0.2, -0.3], [-0.2, 0.3]]"
+        linear_min: 0.0
+        linear_max: 1.0
+        theta_min: -1.0
+        theta_max: 1.0
+      translation_backward:
+        points: "[[0.2, 0.3], [0.2, -0.3], [-0.35, -0.3], [-0.35, 0.3]]"
+        linear_min: -1.0
+        linear_max: 0.0
+        theta_min: -1.0
+        theta_max: 1.0
+      stop:
```

Comment:
`stopped` -> just to be clear its not a "stop"ing polygon, but a polygon to apply when stopped